recipes-core/initramfs-framework/files/0001-Mount-run-with-tmpfs.patch

```diff
@@ -0,0 +1,55 @@
+From 20d74a29c1c9050880f0456b2c3a0242c9c8ab70 Mon Sep 17 00:00:00 2001
+From: Sasi Kumar Maddineni <quic_sasikuma@quicinc.com>
```

Comment:
Since this patch was originally done by me, please have From: Ricardo Salveti <ricardo@foundries.io>